src/options/view.rs

```diff
@@ -350,12 +352,17 @@ impl TimeFormat {
                 //   - there is nothing after `+`
                 // line 1 will be empty when:
                 //   - `+` is followed immediately by `\n`
-                let empty_non_recent_format_msg = "Custom timestamp format is empty, \
-                    please supply a chrono format string after the plus sign.";
-                let non_recent = lines.next().expect(empty_non_recent_format_msg);
+                let invalid_non_recent_format_msg = "Custom timestamp format is invalid, \
+                    please supply a valid chrono format string after the plus sign.";
+                let non_recent = lines.next().expect(invalid_non_recent_format_msg);
                 let non_recent = if non_recent.is_empty() {
-                    panic!("{}", empty_non_recent_format_msg)
+                    panic!("{}", invalid_non_recent_format_msg)
```

Comment:
We'd prefer exiting with an appropriate exit code, like for instance as how is done here:
```rust
                    eprintln!("{e}");
                    trace!("exa.run: exit RUNTIME_ERROR");
                    exit(exits::RUNTIME_ERROR);
```

Reply:
It seems `exits` isn't `pub`, so I'd have to change that. Would it be better to just return an error via `OptionsError`? `OptionsError::Unsupported` seems like a good candidate.